src/main/java/qna/domain/DeleteHistory.java

```diff
@@ -31,6 +31,20 @@ public DeleteHistory(ContentType contentType, Long contentId, User deletedBy, Lo
         this.createDate = createDate;
     }
 
+    private DeleteHistory(ContentType contentType, Long contentId, User deletedBy) {
+        this.contentType = contentType;
+        this.contentId = contentId;
+        this.deletedBy = deletedBy;
+    }
+
+    public static DeleteHistory createQuestionHistory(Question question) {
```

Comment:
메서드 이름이 그대로 읽어보면 "질문 히스토리 생성하기" 인데요.
삭제에 대한 의미도 전달해보는건 어떨까요? 😄 
아래 답변 정적 팩토리 메서드도 동일합니다.
```suggestion
    public static DeleteHistory createDeleteHistoryForQuestion(Question question) {
```